desktop/dashboard.md

```diff
@@ -125,10 +108,99 @@ Click on a specific container for detailed information about the container. The
+
+To pull the latest image from Docker Hub:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Pull**. This pulls the latest version of the image from Docker Hub.
+
+> **Note**
+>
+> The repository must exist on Docker Hub in order to pull the latest version of an image. You must be logged in to pull private images.
+
+### Push an image to Docker Hub
+
+To push an image to Docker Hub:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Push to Hub.**
+
+> **Note**
+>
+> You can only push an image to Docker Hub if the image belongs to your Docker ID or your organization.
+
+### Inspect an image
+
+Inspecting an image displays detailed information about the image such as the image history, image ID, the date the image was created, size of the image, etc. To inspect an image:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Inspect**.
+4. The image inspect view also provides options to pull the latest image, push image to Hub, remove the image, or run the image as a container.
+
+### Remove an image
+
+The **Images** view allows you to remove unwanted images from the disk. The Images on disk status bar displays the number of images and the total disk space used by the images.
+
+You can remove individual images or use the **Clean up** option to delete unused and dangling images.
+
+To remove individual images:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Remove**. This removes the image from your disk.
+
+> **Note**
+>
+> To remove an image used by a running or a stopped container, you must first remove the associated container.
+
+**To remove unused and dangling images:**
+
+An **unused** image is an image which is not used by any running or stopped containers. An image becomes **dangling** when you build a new version of the image with the same tag.
+
+**To remove an unused or a dangling image:**
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your disk.
+2. Select the **Clean up** option from the **Images on disk** status bar.
+3. Use the **Unused** and **Dangling** check boxes to select the type of images you would like to remove.
+
+    The **Clean up** images status bar displays the total space you can reclaim by removing the selected images.
+4. Click **Remove** to confirm.
 
-![Spring app browser view](images/mac-browser-view.png){:width="700px"}
+### Interact with remote repositories
```

Comment:
I know there's not much functionality on the remote repositories tab, but it's key to the whole end-to-end story. So can this section be expanded a bit?

Reply:
Updated